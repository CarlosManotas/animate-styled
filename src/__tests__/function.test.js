import { jsx as _jsx } from "react/jsx-runtime";
import AnimateStyled from "..";
import { render } from "@testing-library/react";
import { test, expect } from "vitest";
import "@testing-library/jest-dom";
test("Component should show 'red' text 'Hello World'", () => {
    const { getByText } = render(_jsx(AnimateStyled, { name: "backInDown", children: undefined }));
    const textElement = getByText("Hello World");
    expect(textElement).toBeInTheDocument();
});
test("Component should match snapshot", () => {
    const { container } = render(_jsx(AnimateStyled, { name: "backInDown", children: undefined }));
    expect(container).toMatchSnapshot();
});
//# sourceMappingURL=function.test.js.map