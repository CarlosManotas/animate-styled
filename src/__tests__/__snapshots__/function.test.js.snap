// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component should match snapshot 1`] = `
<div>
  <div
    backfacevisibility="hidden"
    class="sc-blHHSb bZxzPc"
    delay="0s"
    direction="normal"
    duration="2s"
    fillmode="none"
    iterationcount="infinite"
    name="backInDown"
    opacity="1"
    playstate="running"
    timingfunction="linear"
    transformorigin="center"
  >
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
